tests/components/FilterSearch.test.tsx

```diff
@@ -0,0 +1,295 @@
+import { FilterSearch } from '../../src/components/FilterSearch';
+import { FilterSearchResponse } from '@yext/answers-headless-react';
+import { render, screen } from '@testing-library/react';
+import * as useSynchronizedRequestFunctions from '../../src/hooks/useSynchronizedRequest';
+import userEvent from '@testing-library/user-event';
+import * as searchOperations from '../../src/utils/search-operations';
+import { mockAnswersActions } from '../__utils__/mocks';
+
+jest.mock('@yext/answers-headless-react');
+
+const mockedFilterSearchResponse: FilterSearchResponse = {
```

Comment:
I think we already have some mocked filter search responses in `__fixtures__/data/filtersearch.ts` so maybe we can consolidate these there

Reply:
It's imported by the FilterSearch stories

Reply:
Yes for sure! If anything, FilterSearch.stories also wants the filters, they just weren't strictly necessary

Reply:
refactored the tests to use the old data and added values to the filter fields to the old data

I am unsure about whether the values I added for filter are representative of real data, but I tried to make assumptions based on what I could gather from looking around the code.  I'll leave this unresolved for now and come back to it.